applications/models.py

```diff
@@ -62,8 +62,6 @@ class BerthSwitch(models.Model):
 class BaseApplication(models.Model):
     created_at = models.DateTimeField(verbose_name=_("created at"), auto_now_add=True)
 
-    is_processed = models.BooleanField(verbose_name=_("is processed"), default=False)
```

Comment:
Is this now being checked based on the status?

Reply:
it will be, yes, don't think FE has implemented that just yet